ads/google/a4a/test/test-utils.js

```diff
@@ -161,15 +161,23 @@ describe('Google A4A utils', () => {
 
         url = ['https://foo.com?hello=world', 'https://bar.com?a=b'];
         const config = extractAmpAnalyticsConfig(a4a, headers);
-        const csiRequest = config.requests.visibilityCsi;
-        expect(csiRequest).to.not.be.null;
+        const iniLoadCsiRequest = config.requests.iniLoadCsi;
+        const renderStartCsiRequest = config.requests.renderStartCsi;
+        expect(iniLoadCsiRequest).to.not.be.null;
+        expect(renderStartCsiRequest).to.not.be.null;
         // We expect slotId == null, since no real element is created, and so
         // no slot index is ever set.
-        expect(csiRequest).to.match(new RegExp(
+        expect(iniLoadCsiRequest).to.match(new RegExp(
```

Comment:
Decided to go a different route since expectMatchesAll relies on sinon's expect (which would make it difficult to make it a util function). Let me know if you disagree.